test/dlc_tests/conftest.py

```diff
@@ -326,8 +337,23 @@ def tf_version_within_limit(metafunc_obj, image):
     """
     tf2_requirement_failed = "tf2_only" in metafunc_obj.fixturenames and not is_tf_version("2", image)
     tf24_requirement_failed = "tf24_and_above_only" in metafunc_obj.fixturenames and is_below_tf_version("2.4", image)
+    tf23_requirement_failed = "tf23_and_above_only" in metafunc_obj.fixturenames and is_below_tf_version("2.3", image)
     tf21_requirement_failed = "tf21_and_above_only" in metafunc_obj.fixturenames and is_below_tf_version("2.1", image)
-    if tf2_requirement_failed or tf21_requirement_failed or tf24_requirement_failed:
+    if tf2_requirement_failed or tf21_requirement_failed or tf24_requirement_failed or tf23_requirement_failed:
+        return False
+    return True
+
+
+def mx_version_within_limit(metafunc_obj, image):
```

Comment:
nit: You could merge this function with the one used for tensorflow, and make it generic.

The `image` variable will already contain the framework name to help identify which `is_below_*_version` function should be used.